javalib/src/main/scala/java/io/File.scala

```diff
@@ -41,13 +41,13 @@ class File(_path: String) extends Serializable with Comparable[File] {
   }
 
   def canExecute(): Boolean =
-    Zone { implicit z => access(toCString(path), fcntl.X_OK) == 0 }
+    Zone { implicit z => access(toCString(path), unistd.X_OK) == 0 }
```

Comment:
The change from fcntl.mumble to unistd.mumble in a number of places is a __long__ overdue correction.  Thank you!

Reply:
I fixed everything I saw that was just wrong especially if I had to because of a dependency.